django-stubs/contrib/auth/forms.pyi

```diff
@@ -31,12 +32,42 @@ class UsernameField(forms.CharField):
     def to_python(self, value: Any | None) -> Any | None: ...
     def widget_attrs(self, widget: Widget) -> dict[str, Any]: ...
 
+class SetPasswordMixin:
+    error_messages: dict[str, _StrOrPromise]
```

Comment:
```suggestion
    error_messages: ClassVar[dict[str, _StrOrPromise]]
```

maybe?

Reply:
We don't have this for `error_messages` on other forms and form mixins. I think it's also legitimate to override on a per-instance basis, e.g. change error messages based on the user or type of object.

But I did notice `_ErrorMessagesDict` which I will use here.